integration-tests/README.md

```diff
@@ -48,14 +60,14 @@ tests:
 
 You can use this flag to test in a remote machine too.
 
-## Filtering the tests to run
+### Filtering the tests to run
 
 With the `--filter` flag you can select the tests to run. For instance you can
 pass *MyTestSuite*, *MyTestSuite.FirstCustomTest* or *MyTestSuite.\*CustomTest*:
```

Comment:
I was using *for values* and `for code and commands`. This makes the diff in the website a little crazy, but the markdown seems to be rendered properly.
Would you prefer to use only ` ?